server/src/api/observation/observation.controller.ts

```diff
@@ -1,30 +1,65 @@
 import { Request, Response } from 'express';
-import { getManager } from 'typeorm';
 import { ObservationApiModel } from './observation.types';
 import { Observation } from './observation.model';
 import { ApiRequest, EdipiParam } from '../api.router';
+import { Log } from '../../util/log';
+import { ReportSchema } from '../report-schema/report-schema.model';
+import { RosterInfo, getRostersForIndividual } from '../roster/roster.controller';
 import { timestampColumnTransformer } from '../../util/util';
 
+
 class ObservationController {
   async getAllObservations(req: Request, res: Response) {
     return res.json(await Observation.find());
   }
 
   async createObservation(req: ApiRequest<EdipiParam, ObservationApiModel>, res: Response) {
-    return res.json(await createObservation(req));
+
+    Log.info('Creating observation', req.body);
+
+    const reportingGroup = req.body.reportingGroup;
+    const typeId = req.body.typeId;
+
+    const rosters: RosterInfo[] = await getRostersForIndividual(req.body.edipi, req.body.timestamp.toString(10));
+
+    for (const roster of rosters) {
+      if (hasReportingGroup(roster, reportingGroup)) {
+        const reportSchema = await findReportSchema(typeId, roster.unit.org?.id);
+        if (reportSchema) {
+          return res.json(saveObservationWithReportSchema(req, reportSchema));
+        }
+      }
+    }
+    Log.error(`Unable to save observation from: ${req.body}`);
+    return res.json(new Observation());
+
```

Comment:
```suggestion
```

Reply:
Bad eslint setups?

Reply:
Yeah I'm going to make a pass on our eslint rules here soon to account for these sorts of details.